### PR TITLE
Fixed #2195, improved ConfusionMatrix docs

### LINKED ISSUE #2195: ConfusionMatrix axis - labels
I am looking at the [ConfusionMatrix documentation](https://pytorch.org/ignite/generated/ignite.metrics.confusion_matrix.ConfusionMatrix.html ) and I can't understand what axis corresponds to the _predicted values_ and which one to the _actual values_.

In other words, which one of the two following options is correct?
1) https://miro.medium.com/max/1000/1*fxiTNIgOyvAombPJx5KGeA.png
2) https://miro.medium.com/max/1400/1*85t6zbUiQA0fotnhDaJLaA.png

### PR DESCRIPTION
**Pull request by mfoglio**
Fixes #2195

Description: add information about the format of confusion matrix in the docs

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [x] Documentation is updated (if required)

### CODE REVIEW

File: ignite/metrics/confusion_matrix.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
 class ConfusionMatrix(Metric):
     """Calculates confusion matrix for multi-class data.
 
+    The confusion matrix is formatted such that columns are predictions and rows are targets.
```
Comment (vfdev-5) [inline ignite/metrics/confusion_matrix.py:16]: Let's move this details below after the line L67 and use `Note:` : 

```python
Note:
    The confusion matrix is formatted such that columns are predictions and rows are targets.
    For example, if you were to plot the matrix, you could correctly assign to the horizontal axis
    the label "predicted values" and to the vertical axis the label "actual values".
```

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR @mfoglio 
Looks good, I was just thinking about moving this to the end of the docstring as a note.

File: ignite/metrics/confusion_matrix.py (hunk 1 above)
Comment (mfoglio) [reply ignite/metrics/confusion_matrix.py:16]: Sounds great! @vfdev-5 , should I do the edit? If yes, how? :)

Comment (vfdev-5) [reply ignite/metrics/confusion_matrix.py:16]: Yes, let's edit the file locally as suggested. Next, commit and push: `git commit -a -m "Updated docstring" && git push origin fix-docs-2195`

Comment (mfoglio) [reply ignite/metrics/confusion_matrix.py]: Done. Let me know if that's fine.

Comment (vfdev-5) [reply ignite/metrics/confusion_matrix.py]: Yes, IMO looks good now! Thanks for the PR !

**Review by vfdev-5 — APPROVED** [review]
LGTM

File: ignite/metrics/confusion_matrix.py (hunk 1 above)
Comment (mfoglio) [reply ignite/metrics/confusion_matrix.py]: Thank you for the guidance!

**Pull request merged**